sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_customer_statsbeat_types.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+from enum import Enum
+from typing import Dict, Any, List, Optional, TypedDict
+
+# Language constant for customer statsbeat
+STATSBEAT_LANGUAGE = "python"
+class TelemetryType(str, Enum):
+    AVAILABILITY = "AVAILABILITY"
+    CUSTOM_EVENT = "CUSTOM_EVENT"
+    CUSTOM_METRIC = "CUSTOM_METRIC"
+    DEPENDENCY = "DEPENDENCY"
+    EXCEPTION = "EXCEPTION"
+    PAGE_VIEW = "PAGE_VIEW"
+    PERFORMANCE_COUNTER = "PERFORMANCE_COUNTER"
+    REQUEST = "REQUEST"
+    TRACE = "TRACE"
+    UNKNOWN = "UNKNOWN"
+class DropCode(str, Enum):
+    CLIENT_EXCEPTION = "CLIENT_EXCEPTION"
+    CLIENT_EXPIRED_DATA = "CLIENT_EXPIRED_DATA"
```

Comment:
Remove this due to https://github.com/aep-health-and-standards/Telemetry-Collection-Spec/pull/559